### PR TITLE
chore(js): override eslint config to allow 6 params at max

### PR DESCRIPTION
**Pull request by demonzoo**
## Summary

Override eslint config in project `logto-io/js` to allow 6 function parameters at maximum

## Testing

- [x] Eslint will prompt a warning message when declaring over 6 parameters in any function
image

### CODE REVIEW

**Review by IceHe — APPROVED** [review]
LGTM. (Wait for one more approval)

**Review by xiaoyijun — APPROVED** [review]
LGTM

**Pull request merged**
